data/api/client-server/administrative_contact.yaml

```diff
@@ -120,6 +120,7 @@ paths:
                       description: The client secret used in the session with the identity server.
                     id_server:
                       type: string
+                      format: mx-server-name
```

Comment:
are you sure that `id_server` is supposed to be a server name?

AIUI, synapse accepts things with trailing paths here, which is obviously not allowed by a server name.

See also https://github.com/matrix-org/synapse/pull/9802#discussion_r613172343 which has a bit of historical debate about this. It's certainly questionable whether Synapse *should* allow trailing paths here, but changing it probably needs an MSC.

Reply:
I am not sure of anything, this is what I understood from descriptions like:

> The hostname of the identity server to communicate with. May optionally include a port.

But it is indeed not clear everywhere what this should look like, and looking at the PR you linked it is clear that at least Synapse allows to set a path too.

So I will revert the changes about `id_server`. Should we open an issue that this needs to be clarified?

Reply:
Alright I reverted the `id_server` formats and opened https://github.com/matrix-org/matrix-spec/issues/2056.